srcpkgs/sview/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'sview'
+pkgname=sview
+version=20.08
+revision=1
+wrksrc="${pkgname}-${version//./_}"
+hostmakedepends="libavcodec libavdevice libavformat libavutil libswscale pkg-config"
+makedepends="ffmpeg-devel libopenal-devel libwebp-devel libconfig++-devel libconfig-devel libXpm-devel gtk+-devel libXrandr-devel"
+short_desc="Stereoscopic video player"
+maintainer="Pavel Pivovarov <pavel.pivovarov@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://sview.ru/en"
+distfiles="https://github.com/gkv311/sview/archive/20_08.tar.gz"
+checksum=45029af28b7fb47dba340f6023c70f50b1f696b829831ebdce0a0759ab60652b
+
+do_build() {
+	make FFMPEG_ROOT=${XBPS_CROSS_BASE}/usr
+}
+
+do_install() {
+	DESTDIR=${DESTDIR} make install
+}
```

Comment:
`make_build_args="FFMPEG_ROOT..."` and `build_style=gnu-makefile` should allow this to build without these functions.